src/modules/cdp.ts

```diff
@@ -791,43 +795,53 @@ export class CDPModule extends BaseModule {
     const sdk = this.sdkProvider;
     const network = sdk.getConfig().network;
     const collateralPoolAddress = SWTHAddress.getModuleAddress("collateral_pool", network);
-    const collateralPoolBalances = await sdk.query.bank.AllBalances({ address: collateralPoolAddress });
-
-    const cdpTokenBalances: Coin[] = (collateralPoolBalances?.balances ?? []).filter(balance => tokenClient.isCdpToken(balance.denom))
-    const totalCollateralsUsdValue = (await this.getCdpTokensUsdVal(cdpTokenBalances)) ?? []
-
-    return totalCollateralsUsdValue;
-  }
 
-  public async getCdpTokensUsdVal(cdpTokens: Coin[]) {
-    const sdk = this.sdkProvider;
-    const tokenClient = sdk.getTokenClient();
```

Comment:
Removed this because` Balance()` or `AllBalance()` requires an address to query, but can re-implement this for the general case taking in an address in the form of `getCdpTokensUsdVal(address)`